aiidalab_widgets_base/utils/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 from __future__ import absolute_import
 from __future__ import print_function
+from six.moves import range
```

Comment:
We don't support Py27 anymore so this should not be necessary.

Reply:
ok

Reply:
done